### ISSUE TITLE
feathericons for Adobe XD ?

### ISSUE THREAD

**Issue by YasserDRIF** [feathericons/feather#632]
I'm using feathericons in almost all my designs and till now i love it
But is there any way this to make working with this icon library easier in Adobe Xd

Maybe a resource file or a plugin ?

![image](https://user-images.githubusercontent.com/25182015/60095208-f7f3d300-9745-11e9-97fb-62d424c60f4a.png)

**Comment by mittalyashu** [comment]
Here's my suggestion

| Suggestion |
| --- |
| ![image](https://user-images.githubusercontent.com/29014463/61766287-f0cfeb80-adfd-11e9-98c2-5c75d3bb6f39.png) |

**Comment by locness3** [comment]
Have you read the issue @mittalyashu ??

**Comment by mittalyashu** [comment]
Yes, I have read the issue, but still, we don't have any issue for Adobe @locness3

**Comment by locness3** [comment]
Well, create a seperate one, you're completely off-topic
